src/dynamic-checkout/views/payment-methods.ts

```diff
@@ -264,35 +355,42 @@ module ProcessOut {
       apm: Apm
     ): string {
       return `
-        <div class="dco-pay-button" data-method-id="saved-card" data-card-token-id="${
-          apm.customer_token_id || ""
-        }">
-            <div class="dco-payment-method dco-payment-method--regular" style="background-color: ${DynamicCheckoutColorsUtils.hexToRgba(
-              display.brand_color.dark
-            )}">
-                <img src="${
-                  display.logo.dark_url.vector
-                }" class="dco-payment-method-logo"/>
-                <span class="dco-payment-method-label">${display.name}</span>
-                ${
-                  apm.redirect_url
-                    ? `<span class="dco-payment-method-apm-message">You will be redirected to finalise this payment</span>`
-                    : ""
-                }
-            </div>
-        </div>`;
+        <div
+          class="dco-pay-button"
+          data-method-id="saved-card"
+          data-card-token-id="${apm.customer_token_id || ""}"
+        >
+          <div class="dco-payment-methoddco-payment-method--regular">
+              <img
+                src="${display.logo.dark_url.vector}"
+                class="dco-payment-method-logo"
+              />
+              <span class="dco-payment-method-label">${display.name}</span>
+              ${
+                apm.redirect_url
+                  ? `<span class="dco-payment-method-apm-message">You will be redirected to finalise this payment</span>`
```

Comment:
nit: I remember someone saying we should use en_US, not en_GB, but I'm not 100% sure this is a company-wide policy.
```suggestion
                  ? `<span class="dco-payment-method-apm-message">You will be redirected to finalize this payment</span>`
```